src/components/SkipModalWindow/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core/';
+
+const SkipModalWindow = (props) => {
+  const { active = false, onClose, fetchTestAnswers } = props;
+
+  return (
+    <Dialog
+      open={active}
+      onClose={onClose}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+    >
+      <DialogTitle id="alert-dialog-title">{'Skip question'}</DialogTitle>
```

Comment:
Why do you need to use JS expression here?

Reply:
Fixed. Update in next commit